docs/source/courses/software-skills.md

```diff
@@ -0,0 +1,119 @@
+# General software skills for systems neuroscience
+
+## Overview
+Held over seven days, this course will provide researchers with some basic computational skills for systems 
+neuroscience research. This will include general programming, software development best practices and data analysis 
+using leading open-source software tools. This course will also include some general microscopy lectures. 
+
+## Schedule
+### Monday, September 30th
+- 13:00-14:00: Introduction to the course (SWC Brasserie Seminar Room)
+- 14:00-17:00: Introduction to Python (1) (SWC Brasserie Seminar Room)
+
+### Tuesday, October 1st
+- 10:00-11:00: Data management and sharing (1) (SWC Ground Floor Lecture Theatre)
+- 15:00-17:00: Data management and sharing (2) (SWC Ground Floor Lecture Theatre)
+
+### Wednesday, October 2nd
+- 10:00-12:30: Introduction to Python (2) (GCNU Seminar Room)
+- 13:30-17:00: Version control and software development best practices (GCNU Seminar Room)
+
+### Thursday, October 3rd
+- 10:00-13:00: Video-based analysis of animal behaviour (1) (SWC Brasserie Seminar Room)
+- 14:00-17:00: Video-based analysis of animal behaviour (2) (SWC Brasserie Seminar Room)
+
+### Friday, October 4th
+- 14:00-17:00: Linux and high-performance computing (SWC Ground Floor Lecture Theatre)
+
+### Monday, October 7th
+- 11:00-12:00: General microscopy: basics (SWC Brasserie Seminar Room)
+- 13:00-14:30: General microscopy: applications (SWC Brasserie Seminar Room)
+- 14:30-18:00: Histology analysis (1) (SWC Brasserie Seminar Room)
+
+### Tuesday, October 8th
+- 10:00-11:00: Histology analysis (2) (SWC Ground Floor Lecture Theatre)
+- 14:00-17:00: Histology analysis (3)  (SWC Brasserie Seminar Room)
```

Comment:
I personally think the schedule would be much more readable as a table.
```suggestion
| Date                  | Time           | Event                                                        | Location                           |
|-----------------------|----------------|--------------------------------------------------------------|------------------------------------|
| **Monday, September 30th** | 13:00-14:00    | Introduction to the course                                    | SWC Brasserie Seminar Room         |
|                       | 14:00-17:00    | Introduction to Python (1)                                    | SWC Brasserie Seminar Room         |
| **Tuesday, October 1st**   | 10:00-11:00    | Data management and sharing (1)                                | SWC Ground Floor Lecture Theatre   |
|                       | 15:00-17:00    | Data management and sharing (2)                                | SWC Ground Floor Lecture Theatre   |
| **Wednesday, October 2nd** | 10:00-12:30    | Introduction to Python (2)                                    | GCNU Seminar Room                  |
|                       | 13:30-17:00    | Version control and software development best practices       | GCNU Seminar Room                  |
| **Thursday, October 3rd**  | 10:00-13:00    | Video-based analysis of animal behaviour (1)                  | SWC Brasserie Seminar Room         |
|                       | 14:00-17:00    | Video-based analysis of animal behaviour (2)                  | SWC Brasserie Seminar Room         |
| **Friday, October 4th**    | 14:00-17:00    | Linux and high-performance computing                          | SWC Ground Floor Lecture Theatre   |
| **Monday, October 7th**    | 11:00-12:00    | General microscopy: basics                                    | SWC Brasserie Seminar Room         |
|                       | 13:00-14:30    | General microscopy: applications                              | SWC Brasserie Seminar Room         |
|                       | 14:30-18:00    | Histology analysis (1)                                        | SWC Brasserie Seminar Room         |
| **Tuesday, October 8th**   | 10:00-11:00    | Histology analysis (2)                                        | SWC Ground Floor Lecture Theatre   |
|                       | 14:00-17:00    | Histology analysis (3)                                        | SWC Brasserie Seminar Room         |
```

Reply:
Great, thanks!